task/10-katas-1-tasks.js

```diff
@@ -113,7 +184,45 @@ function getZigZagMatrix(n) {
  *
  */
 function canDominoesMakeRow(dominoes) {
-    throw new Error('Not implemented');
+    let temp = dominoes.shift(),
+         i = 0;
+    while (dominoes.length != 0)
```

Comment:
rework to `while (dominoes.length)`

Please, read about https://developer.mozilla.org/en-US/docs/Glossary/Truthy